src/Components/Error.jsx

```diff
@@ -0,0 +1,8 @@
+export const Error = () => { 
+    return (
+      <div className="error">
+        <h1>Error 😥</h1>
```

Comment:
a11y wise, it would be good to have a span tag wrapped round any emojis with some alt text (just something to consider)